domain/src/main/assets/fractions_exploration1.json

```diff
@@ -5304,7 +5304,7 @@
               "param_changes": [],
               "feedback": {
                 "content_id": "feedback_6",
-                "html": "<p>No, that's not right. It looks like you've forgotten what the numerator and denominator represent! The denominator should be the <strong>total</strong> <strong>number of equal parts</strong>, but it isn't in your answer.</p><p>You should probably try the <oppia-noninteractive-link text-with-value=\"&amp;quot;refresher lesson&amp;quot;\" url-with-value=\"&amp;quot;https://www.oppia.org/explore/xHwiGtvK2N4L?parent=MjZzEVOG47_1&amp;quot;\"></oppia-noninteractive-link> to refresh your memory.</p>"
+                "html": "<p>No, that's not right. It looks like you've forgotten what the numerator and denominator represent! The denominator should be the <strong>total</strong> <strong>number of equal parts</strong>, but it isn't in your answer.</p><p>You should probably try the <oppia-concept-card-link skill-id=\"UxTGIJqaHMLa\">refresher lesson</oppia-concept-card-link> to refresh your memory.</p>"
```

Comment:
Replace last sentence with: If you like, you can take a look at this [concept card] to refresh your memory.

Reply:
Done.